internal/epoch/epoch_manager.go

```diff
@@ -624,6 +643,24 @@ func (e *Manager) maybeGenerateNextRangeCheckpointAsync(ctx context.Context, cs
 	})
 }
 
+func getRangeToCompact(cs CurrentSnapshot, p Parameters) (low, high int, compactRange bool) {
+	latestSettled := cs.WriteEpoch - numUnsettledEpochs
```

Comment:
(minor readability suggestion) I realize this is just a refactor of existing logic, but perhaps we can make a new helper
```go
(cs *CurrentSnapshot) getLastSettledEpochNumber() int {
   return cs.WriteEpoch - numUnsettledEpochs
}
```
and we can reuse that here, as well as in `cs.isSettledEpochNumber()`?
```go
func (cs *CurrentSnapshot) isSettledEpochNumber(epoch int) bool {
	return epoch <= cs.getLastSettledEpochNumber()
}
```
For me it's easier to interpret the significance of that subtraction operation if it's described by a helper function name 🤷

Reply:
Makes sense. It can be done in a separate PR.